cst438-homework1/src/main/resources/templates/movie_form.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html xmlns:th="www.thymeleaf.org">
+<head>
+<meta charset="UTF-8">
+<title>Movie Form</title>
+</head>
+<body>
+	<h2>Rate a Movie!</h2>
+	<form action="#" th:action="@{/movies/new}" th:object="${movieRating}" method="post">
+	Reviewer's name: <input type="text" th:field="*{reviewerName}" /> 
+		<span th:if="${#fields.hasErrors('reviewerName')}" th:errors="*{reviewerName}"> </span> <br />
+	Movie Title: <input type="text" th:field="*{title}" />
+	    <span th:if="${#fields.hasErrors('title')}" th:errors="*{title}"> </span> <br /> 
+	Rating (1-5): <input type="number" min="1" max="5" th:field="*{rating}" /> 
```

Comment:
This is min and max handling is good here, but don't forget to also add validation on the backend.